units/Legion/Economy/legmohoconct.lua

```diff
@@ -0,0 +1,89 @@
+return {						--costs should be same as legmohocon and legmohoconin
+	legmohoconct = {
+		maxacc = 0,
+		maxdec = 4.5,
+		energycost = 8100,
+		metalcost = 640,
+		builddistance = 400,
+		builder = true,
+		buildpic = "LEGMOHOCON.DDS",
+		buildtime = 14100,
+		canassist = true,
+		canfight = true,
+		canguard = true,
+		canpatrol = true,
+		canreclaim = true,
+		canrepeat = false,
+		canstop = true,
+		cantbetransported = false,
+		collisionvolumeoffsets = "0 0 0",
+		collisionvolumescales = "31 32 31",
+		collisionvolumetype = "CylY",
+		usePieceCollisionVolumes = true,
+		energyupkeep = 30,
+		explodeas = "largeBuildingexplosiongeneric",
+		footprintx = 4,
+		footprintz = 4,
+		idleautoheal = 5,
+		idletime = 1800,
+		mass = 700,
+		health = 3900,
+		maxslope = 10,
+		maxwaterdepth = 0,
+		stealth = true,
+		movementclass = "NANO",
+		objectname = "Units/LEGMOHOCON.s3o",
+		script = "Units/legmohoconct.cob",
+		seismicsignature = 0,
+		selfdestructas = "largeBuildingExplosionGenericSelfd",
+		sightdistance = 380,
+		terraformspeed = 1000,
+		turnrate = 1,
+		upright = true,
+		workertime = 400,
+		icontype = "cormoho",
+		customparams = {
+			usebuildinggrounddecal = true,
+			buildinggrounddecaltype = "decals/cornanotc_aoplane.dds",
+			buildinggrounddecalsizey = 5,
+			buildinggrounddecalsizex = 5,
+			buildinggrounddecaldecayspeed = 30,
+			unitgroup = 'builder',
+			model_author = "Tharsis and Protar",
+			normaltex = "unittextures/leg_normal.dds",
+			subfolder = "CorBuildings/LandUtil",
```

Comment:
not a cor building, tho i don't know what purpose this servers.

Reply:
Removed.